lib/index.ts

```diff
@@ -1,4 +1,9 @@
-export enum Type {
+import { generateOcsUrl } from "@nextcloud/router";
+import axios from '@nextcloud/axios'
+
+import type { AxiosResponse } from 'axios'
+
+export enum NcShareType {
```

Comment:
To prevent naming conflicts in libraries that use it

Reply:
Agree that `Type` is very generic, but `ShareType` should be good.
If it conflicts then just import it with alias like `import { ShareType as NcShareType } from '...'`.

Especially as we do not use prefixes on e.g. nextcloud-files or all other libraries